cms/templates/partials/nhs_header.html

```diff
@@ -0,0 +1,36 @@
+{% load frontend_tags menu_tags %}
```

Comment:
Confused by this -- doesn't seem to be relevant to the breadcrumbs -- doesn't obviously seem to refer or be referred to by the breadcrumbs bits.

Reply:
I've fixed that. I'm not sure why it shoeing as new. A git blame points to some bloke called McKee. :)

Reply:
The merge adds it -- trying to rebase instead gets the following conflict:

```
CONFLICT (rename/rename): cms/templates/partials/header-v1.html renamed to cms/templates/partials/nhs_components/nhs_header.html in HEAD and to cms/templates/partials/nhs_header.html in e3598d1 (Add in partials into search template and general page styling.).
```

My suspicion is that we'll wind up with two copies of nhs_header as this PR currently stands

Reply:
I think you're right. Israt moved the files, I'll tidy up the higher file